net-misc/balance/balance-3.57-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps toolchain-funcs
+
+DESCRIPTION="TCP Load Balancing Port Forwarder"
+HOMEPAGE="http://www.inlab.de/balance.html"
+SRC_URI="http://www.inlab.de/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
+
+PATCHES=( "${FILESDIR}"/${P}-Makefile.patch )
+
+FILECAPS=(
+	'CAP_NET_BIND_SERVICE' '/usr/sbin/balance'
+)
+
+src_prepare() {
+	default
+
+	tc-export CC
+}
+
+src_install() {
+	default
+
+	#autocreated on program start, if missing
+	rm -rv "${ED}/var/run" || die
+}
+
+pkg_postinst() {
+	fcaps_pkg_postinst
+	elog "To run as non-root, be sure to have rendezvous directory created"
+	elog "with either 'mkdir -m 01777 /var/run/balance' or using tmpfiles."
```

Comment:
I personally don't think this message is necessary here, because `balance` itself prints following message on start:
```
ERROR: rendezvous directory not available and/or creatable
       please create /var/run/balance/ with mode 01777 like this: 
       # mkdir -m 01777 /var/run/balance/
```
But it is probably ok to go with it.

Reply:
In case `/var/run` is on tmpfs and cleaned with reboots, this will point the user on what tool (tmpfiles) to use for solving this problem correctly and what's required for the server to run.